crates/sui-core/src/consensus_handler.rs

```diff
@@ -114,6 +128,14 @@ impl<T: ParentSync + Send + Sync> ExecutionState for ConsensusHandler<T> {
             Arc::new(consensus_output.sub_dag.leader.clone()),
         ));
 
+        // TODO: spawn a separate task for this as an optimization
+        update_low_scoring_authorities(
+            &self.low_scoring_authorities,
+            self.committee.clone(),
```

Comment:
That change has landed, I rebased on top of it, it still is cloneable and needs to be, but just not cloned on the critical path. Once per epoch should be fine, and is unavoidable here since we pass this committee to be saved on a struct, it cant be a reference.